src/app/pages/AccountPage/index.tsx

```diff
@@ -213,3 +214,15 @@ export function AccountPage(props: Props) {
     </Box>
   )
 }
+
+export function AccountPage(props: AccountPageProps) {
+  const { t } = useTranslation()
+  const { address } = useParams<keyof AccountPageParams>()
+  return isValidAddress(address!) ? (
+    <AccountPageInternal {...props} />
+  ) : (
+    <Box pad={'small'}>
```

Comment:
(I have slight style preference to have general case as the last return, and special cases as early returns)
```ts
  if (!isValidAddress(address!)) {
    return <Box pad={'small'}> ..
  }

  return <AccountPageInternal {...props} />
```

Reply:
I agree. Changed accordingly.